templates/home/index.html.twig

```diff
@@ -1,13 +1,13 @@
 {% extends 'base.html.twig' %}
 {% block body %}
 <body>
-    <style> #mapid { height: 750px; }
-        #logo{height: 150px;}
-        a{ align-items: center; display: flex; justify-content: space-between;}
-    </style>
     <img id="logo" src="{{ asset('/build/images/logo.52dc2bfe.png') }}" alt=""><br>
-    <a href="{{ path('country') }}">Country</a></br>
-    <a href="{{ path('university') }}">University</a>
+    <hr>
```

Comment:
from [documentation](see https://developer.mozilla.org/fr/docs/Web/HTML/Element/hr):

> The HTML \<hr\> element represents a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section.

Your hr isn't between paragraph-level elements: did you try to display an horizontal line? If that's the case, apply a border in css to the previous or to the next element